lib/voyager_helpers/liberator.rb

```diff
@@ -192,6 +192,7 @@ def get_availability(bibs, full=false)
               else
                 item = get_info_for_item(holding_item_ids.first, c, false)
                 availability[bib_id][mfhd_id][:temp_loc] = item[:temp_location] unless item[:temp_location].nil?
+                availability[bib_id][mfhd_id][:course_reserves] = get_courses(holding_item_ids, c).map(&:to_h)
```

Comment:
One way we can prevent the get_courses query from being triggered is only to fire it when `full` is true? `full` is only true when a single bib id is passed to the get_availability method.
https://github.com/pulibrary/voyager_helpers/blob/7e37784462419d81fded596b7f812eee7869b16f/lib/voyager_helpers/liberator.rb#L167